forge-gui/res/cardsfolder/upcoming/last_night_together.txt

```diff
@@ -0,0 +1,12 @@
+Name:Last Night Together
+ManaCost:3 R G
+Types:Sorcery
+A:SP$ Untap | ValidTgts$ Creature | TgtPrompt$ Choose two target creatures | RememberTargets$ True | TargetMin$ 2 | TargetMax$ 2 | TargetUnique$ True | Defined$ Targeted | SubAbility$ DBPutCounter | SpellDescription$ Choose two target creatures. Untap them. 
+SVar:DBPutCounter:DB$ PutCounter | Defined$ Targeted | CounterType$ P1P1 | CounterNum$ 2 | SubAbility$ DBPump | Put two +1/+1 counters on each of them. 
+SVar:DBPump:DB$ Pump | Defined$ Targeted | KW$ Vigilance & Indestructible & Haste | SubAbility$ DBAddCombat | SpellDescription$ They gain vigilance, indestructible, and haste until end of turn.
+SVar:DBAddCombat:DB$ AddPhase | ExtraPhase$ Combat | FollowedBy$ Main2 | ExtraPhaseDelayedTrigger$ DelTrigStatic | ExtraPhaseDelayedTriggerExcute$ TrigEffect | SpellDescription$ After this main phase, there is an additional combat phase. Only the chosen creatures can attack during that combat phase.
+SVar:DelTrigStatic:Mode$ Phase | Phase$ BeginCombat | TriggerDescription$  After this main phase, there is an additional combat phase. Only the chosen creatures can attack during that combat phase.
+SVar:TrigEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ ForbidAttack | Duration$ UntilEndOfCombat
```

Comment:
The remembered would stick around?
maybe something like this works instead: `Spawner>Targeted`

Reply:
Hmm this version doesn't work. 

Woudn't a cleanup work ?

Reply:
If you can figure out how to clean it up directly so it doesn't end up with Remembered in graveyard...

Unfortunately my trick doesn't work because the spawningAbility targets are done on a parent which aren't copied and therefore get cleared on resolve
I'd need to think on that some more ⚙️ 

What could work is if you use ChooseCard first like you wanted to with Pump:
- the Effect should pick it up by default
- it doesn't require cleanup anyway
- card is templated a bit like it as well (even if it's only to improve readability)

Reply:
I tried it this way but the effects doesn't seem to be present anymore (no extra phase as well 🤔 )